requests/Journal.ts

```diff
@@ -0,0 +1,60 @@
+// these functions call the /pages/api/... endpoints
+import fetch from "isomorphic-unfetch";
+import { ContentfulImage, JournalEntry } from 'utils/types';
+
+// TODO: create api endpoint for paginated entries so we dont have
+// to pull all entries and process on the front end. fine for now
+// though.
+
+// assuming we only want to show reviewed entries by default
+export const getEntries = () =>
+  fetch("http://localhost:3000/api/journal/getByReviewStatus?reviewed=true", {
+    method: "GET",
+    mode: "same-origin",
+    headers: {
+      "Content-Type": "application/json",
+    },
+  })
+  .then((response) => response.json())
+  .then((data) => {
+    if (!data) {
+      throw new Error("Could not connect to API!");
+    }
+    return data;
+  });
+
+export const getEntriesByType = async (type: any) => {
```

Comment:
Forgot to mention, this method doesn't only get reviewed posts, so when sorting by a category it may show unreviewed content.

Reply:
Shoot you're right. We need to add `reviewed: true` to the query parameter in Contentful.